packages/router/src/grpc/custody/authorize.ts

```diff
@@ -26,7 +26,9 @@ export const authorize: Impl['authorize'] = async (req, ctx) => {
   if (!decryptedSeedPhrase)
     throw new ConnectError('Unable to decrypt seed phrase with password', Code.Unauthenticated);
 
-  await approveReq(req);
+  await getTxApproval(req).catch(no => {
+    throw ConnectError.from(no, Code.PermissionDenied);
+  });
```

Comment:
Is that required? Why can't we just let its own error bubble up?

Reply:
throwing as connecterror provides some informative detail codes. and until #346 it won't bubble otherwise

Reply:
merged, but i'd like to keep this as the error code adds detail